packages/lesmis/src/routes/AdminPanelRoutes.js

```diff
@@ -128,22 +132,12 @@ export const ROUTES = [
       {
         label: 'Users',
         to: '',
-        component: UsersPage,
+        component: UsersView,
       },
       {
         label: 'Permissions',
         to: '/permissions',
-        component: PermissionsPage,
-      },
-      {
-        label: 'Permission Groups',
-        to: '/permission-groups',
-        component: PermissionGroupsPage,
```

Comment:
remove permission groups as they are a defined list on lesmis and shouldn't be edited.